### ISSUE TITLE
Is there is a provision to run the performance tests using command line cli like newman or postman cli ?

### ISSUE THREAD

**Issue by sketha** [a85/POSTMan-Chrome-Extension#12008]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Currently postman has the provision to run the collections using the newman cli or the postman cli, can we also have the provision to run the performance collections using the command cli to enable to the CI/CD.

### Describe the solution you'd like

Export the collection and pass the necessary parameters as command line arguments and then initiate the test.

**Comment by malvikach** [comment]
Hi Sunil,

Thanks for raising this feature request. We have taken note of this and will keep you updated.  

As discussed, Automating Performance Tests is something we are exploring and we will have more information to share in months to come.

Thanks, 
Malvika

**Comment by mudit94** [comment]
Hi Team, Is there any update on this?

**Comment by DrGazian** [comment]
Yeah, also looking for update, would like to be able to use Postman to performance test on an EC2 instance, seems like the feature might be slightly pointless - due to latency issues etc - if unable to utilise this way

**Comment by mpp-seanc** [comment]
@malvikach, we're also interested in this enhancement for automating postman performance testing via CLI.  Is there any hint of this being on a roadmap?

**Comment by craigd-lumin** [comment]
Hi @malvikach, I'd also be grateful for any update on this feature!?

**Comment by rghv932** [comment]
Hi @malvikach 
Would like to know any update on this?

**Comment by tnielsen2** [comment]
Having the ability to run a performance test with Postman CLI on a remote machine with automation is something that we want and would use. Any update on this?

**Comment by JvBPropel** [comment]
Bump

**Comment by KokWeiGan** [comment]
Up Up
We need this

**Comment by juris-kreilis** [comment]
Please consider this feature, we need this as well.

**Comment by KokWeiGan** [comment]
Hi ,@malvikach 

Any update about this feature?

**Comment by dairiskk** [comment]
Hi @malvikach When its planned to add this feature?

**Comment by rafcio19** [comment]
+1 to this feature request

**Comment by EthraZa** [comment]
+1 ( the 1st in 2025 =)

**Comment by cah-manik-vashisht01** [comment]
+2

**Comment by abhishek-kandala** [comment]
+1 Is there any update on this @malvikach ?

**Comment by shamasis** [comment]
Hello folks. This is out in beta https://github.com/postmanlabs/postman-app-support/issues/13946

<img width="930" height="604" alt="Image" src="https://github.com/user-attachments/assets/b5e7bfc3-54c0-4250-afde-ee441a1a1a01" />


```terminal
# Install Postman CLI
npm install postman-cli@latest -g

# Run performance test
postman performance run <collection-id> --vu-count 50 --duration 15
```

**Comment by saialekhya-001** [comment]
Update for those wanting to run load from something other than a local machine: cloud performance runs are now available (beta). Instead of standing up an EC2 box yourself, Postman runs the test across multiple machines in its own managed infrastructure, so load generation is no longer tied to your laptop's resources or network. You can start a cloud run from the Performance tab in the Collection Runner (desktop app). 
For CI/CD-triggered runs, the Postman CLI is available as mentioned — but if the goal is to scale without managing your own infra, cloud runs is the way forward.

<img width="1512" height="543" alt="Image" src="https://github.com/user-attachments/assets/2739b0c7-1a97-46da-9f1b-96358719a049" />
